GeorgianMemenator/Scenes/DogBreedsList/DogBreedsListView.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  DogBreedsListView.swift
+//  GeorgianMemenator
+//
+//  Created by Qeti Avanashvili on 08.01.23.
+//  Copyright © 2023 Shalva Avanashvili. All rights reserved.
+//
+
+import SwiftUI
+import Foundation
+
+struct DogBreedsListView: View {
+    var body: some View {
+        EmptyView()
+    }
+}
+
+struct DogBreedsListView_Previews: PreviewProvider {
+    static var previews: some View {
```

Comment:
Whenever you will create class `DogBreedsListViewModel`, we will need viewmodel property here.

`@ObservedObject var viewModel: DogBreedsListViewModel`